tests/unit/store/flask.spec.js

```diff
@@ -291,7 +291,14 @@ describe("store/flask", () => {
           STATUS.MESSAGE.ERROR
         );
       });
+      test("Given the status is set to ERROR, When attempting to commit SHUTDOWN, Then it remains in updates to SHUTDOWN mode", () => {
```

Comment:
this looks like maybe a typo: `remains in updates`

Did you mean to just say `updates`?

Reply:
Yes, I missed to correct them will do the same.